dirs/dirs.go

```diff
@@ -345,6 +433,7 @@ func SetRootDir(rootdir string) {
 	if rootdir == "" {
 		rootdir = "/"
 	}
+	filepath.Clean(rootdir)
```

Comment:
What was this expected to do? It is not modifying `rootdir`. This looks like a leftover or like a bug (more likely?).